jsv/features/homepage-lomita.feature

```diff
@@ -10,6 +10,7 @@ Feature: Lomita Homepage Layout
   #  Given the "stanford_jumpstart_home_lomita" homepage layout is not selected
     Given I am logged in as a user with the "administrator" role
     And I am on "admin/stanford-jumpstart/customize-design"
+    Then I press the "edit-layouts-stanford-jumpstart-home-panama-selector" button
```

Comment:
This line is going to change slightly again as the products use different permissions for the site owner role. I am going to generalize it more to pick the first available button or something similar.